Development/src/pages/devices/ChannelMappingMatrix.js

```diff
@@ -0,0 +1,514 @@
+import { useState } from 'react';
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TableRow,
+    Tooltip,
+    Typography,
+} from '@material-ui/core';
+import {
+    ReferenceField,
+    ReferenceManyField,
+    SingleFieldList,
+} from 'react-admin';
+import get from 'lodash/get';
+import ChipConditionalLabel from '../../components/ChipConditionalLabel';
+import MappingButton from '../../components/MappingButton';
+import CollapseButton from '../../components/CollapseButton';
+
+const get_output_caps = (output_item, io) => {
+    return (
+        <>
+            <Typography variant="body1">{'Description:'}</Typography>
+            {output_item.properties.description}
+            <Typography variant="body1">{'Routable inputs:'}</Typography>
+            {output_item.caps.routable_inputs !== null
+                ? output_item.caps.routable_inputs
+                      .map(input_name => {
+                          return input_name === null
+                              ? 'MUTE'
```

Comment:
A place to think about NMOS terminology vs user friendly terms 🤔 MUTE vs unrouted vs... not sure!

Reply:
(We'd often use the ✖️ icon for this kind of `null` but that doesn't play nicely as part of the list of inputs I expect)

Reply:
We discussed in call today, maybe leaving as MUTE (or MUTED better?), though the IS-08 spec doesn't use that word, instead "unrouted" or "parked", "silence", or just `null`.

> An Output channel that does not have an Input channel routed to it is said to be "unrouted" (sometimes known as "parked")
([link](https://amwa-tv.github.io/nmos-audio-channel-mapping/tags/v1.0.1/docs/1.0._Overview.html#terminology))

> If no Input channel is being routed to an Output channel, both `input` and `channel_index` MUST be set to `null`. Where an Output does not have a channel routed to it the Device MUST interpret this as silence, and produce a silent signal of appropriate bit depth and bit rate from that output.
([link](https://amwa-tv.github.io/nmos-audio-channel-mapping/tags/v1.0.1/docs/4.0._Behaviour.html#the-active-map))

Reply:
I will commit a change to use 'Unrouted' rather than 'MUTE' for this first PR, and agreed with @rhastie that we can change depending on initial demo feedback.